### ISSUE TITLE
List of updates / feedback which need to be incorporated in "Onboarding" application

### ISSUE THREAD

**Issue by MustafaASK** [ShaikASK/Testing#469]
**Feedback from David Team -**
 
•	Job ID - Is this Accuick ID or Client Provided Job ID?. If Client provided Job ID it should be optional. [Shaik: It is a Client provided Job ID]
 
•	Additional field 'Legal Name' to be added?  Also ‘Middle Name’ should be part of this New Hire screen? – To be discussed with Manish
 
•	Bill Rate/OT Bill Rate - Error message to be updated "Bill Rate should be higher than Pay Rate" - Done
 
•	OT Rate label update with OT Pay Rate - Done
 
•	OT Rate should be higher than Pay Rate - Done
 
•	Additional Pay rate Fields to be added  – To be discussed with Manish
Double Time Rate
Double Time Bill Rate
 
•	Additional Address Fields to be added – To be discussed with Manish
Client Postal Address
Candidate Postal Address
 
•	Sub Vendor E-mail should allow multiple E-mail IDs?– To be discussed with Manish
 
•	Additional field 'Category' to be added for ‘IT or Non IT’?- – To be discussed with Manish
 
 
**Feedback from Sagar Team -**
 
Job ID should take special character '-'. This Job ID column can be optional. – This will be done
With the same Job ID we should have multiple Candidates. – Already Covered
 
Additional Field 'Duration of the Contract' is required? This duration needs to be part of the Offer Letter
Additional Field 'Sourcer Name' to be added? They will also get the benefit along with Recruiter
Additional Field 'Mileage Reimbursement' check box to be added?
Additional Field 'Tentative end date'
‘Reports’ module should be accessible to the Account Managers? Currently it accessible ONLY to HR teams.

**Comment by MustafaASK** [comment]
Hi Manish,
As discussed on last Saturday, below are the updates on the feedback received from Accounts, and HR teams.
David/Sagar/Jimmy – All of you please go through the below updates and let us know in case of any queries/concerns.
Update on the Feedback received from Account Managers
1.      Job ID - Is this Accuick ID or Client Provided Job ID?. If Client provided Job ID it should be optional. [Shaik: It is a Client provided Job ID]
Provide Accuick ID for the Job ID
2.      Additional field 'Legal Name' to be added?  Also ‘Middle Name’ should be part of this New Hire screen?
Add Middle Name column. Along with First Name, Middle Name, and Last Name this will become a Legal Name. Middle Name can be optional.
3.      Bill Rate/OT Bill Rate - Error message to be updated "Bill Rate should be higher than Pay Rate"
Already Done
4.      OT Rate label update with OT Pay Rate
Already Done
5.      OT Rate should be higher than Pay Rate
Already Done
6.      Additional Pay rate Fields to be added - 
a.      Double Time Rate
b.      Double Time Bill Rate
These columns will be added Later. For the time being these details can be mentioned in the Notes text area.
7.      Additional Address Fields to be added -
a.      Client Postal Address – Add this files as ‘Client Street Address’. It is mandatory.
b.      Candidate Postal Address –  For the time being this Candidate address can be mentioned in the  ‘Notes’ text area.
8.      Sub Vendor E-mail should allow multiple E-mail IDs?  Any additional e-mail IDs can enter in the ‘Notes’ column
9.      Additional field 'Category' to be added for IT or Non IT? – Add Combo box for this ‘Job Category’ as part of the current prod. release. Values ‘IT’, ‘Non IT’,… should come from the DB. It is mandatory field.
10.     Job ID should take special character '-'. This Job ID column can be optional. With the same Job ID we should have multiple Candidates.
Already Done.
11.     Additional Field 'Duration of the Contract' is required? This duration needs to be part of the Offer Letter – ‘Duration of the Contract’ field to be added in the current prod. release.
12.     Additional Field 'Sourcer Name' to be added? They will also get the benefit along with Recruiter – These ‘Sourcer’ columns will can added latest after post production. These values will be optional.
13.     Additional Field 'Mileage Reimbursement' check box to be added? – Can be added in next release.
14.     Additional Field 'Tentative end date' – This column to be added in the current prod. release
15.     ‘Reports’ module should be accessible to the Account Managers? Currently it accessible ONLY to HR teams. – This will be done in next release.
Update over Key Changes or Additions discussed -
1.      Dashboard screen – In Recent Activities section do we need to display login/logout activities? –  As of now we are continuing with this. Later we can improve this experience by studying the data.
2.      General Settings screen – Offer Letter reminder notification SLA Hours are not required for some New Hires. For this we need to add ‘Not send reminder notification’  checkbox.  Also HR user should also be able to update this setting when needed. – From General settings set ‘Offer Letter response hours’ to 0 for NOT sending any reminder e-mails to all the relevant candidates from batch job.
3.      Revise offer letter – Even after Candidate accepted the offer letter and while workflow steps are in progress, HR needs the ability to send the ‘Revised offer letter’ without impacting any other work flow step/document statuses. – This will be done in the next release. Currently send it manually.
4.      Upload Document screen - Add one more setting name ‘Client Specifications’ in document settings (along with BGV, EEO,..) – Add it in the current release itself.
5.      Candidate Web forms – After filling each Web forms, provide the for Preview Web Form data in PDF format. So that candidate can preview these details in Adobe.  – We will think and plan it in the next release.
6.      Candidate Dashboard – Provide the ‘EDIT’ option to the candidate while reviewing/signing the PDF document. This is required in case Candidate sees some details inside PDF to be corrected. Currently after all the Web forms are filled, Candidate can’t edit these details while reviewing/signing the PDF documents. This edit details is possible only after the HR is rejected the signed document as part of their Review.
-       Allow the ‘Edit of the Web forms’ till Candidate not Signed ANY document. – This needs to be done in the current release.
-       In case few document ‘Signed’ by the Candidate, and still Candidate can modify the data using Web forms. Here we will display the POP up that they need to re-sign the relevant PDF documents. This will be done in the next release.
7.      Candidate Dashboard – Against BGV PDF document, adding supporting files is NOT mandatory, make it as optional.  – HR can achieve this by selecting ONLY ‘HR Certification’ with ‘BGV’ while uploading documents. No changes are required.
8.      HR Review & Certify screen – For Data Protection, currently all the secured details (like SSN, Driving License#,…) are encrypted inside the signed PDF document. A separate icon is provided to HR users to view 
these secured candidate details. However HR can’t share these secure details to Vendor finance teams directly through PDF documents as they use to do manually. 
Add a new role for Vendor finance teams (like ADP) who can log into our portal ONLY to view these Secured Details. 
Display the secure information (without encrypting) as is in the PDF document. This needs to be done in the current release.
Remove the displaying of secure details from the HR Review & Certify screen.
9.      Work flow creation - Steps without Documents (Drug Test, and I9 verification)
This is already done.
10.     For C2C/1099 there should NOT be any offer letter. PO will be send to the Candidate after Onboarding process is successfully completed. 
PO should be outside of Onboarding. This will be achieved by creating a ‘C2C offer letter’ template.

**Comment by MustafaASK** [comment]
24/07/2018 - Natasha Chouhan - We have retested this on the latest Release#3 Build 14 and observed that this is working as expected , Hence marking this to close
